docs/api-reference/carto/overview.md

```diff
@@ -54,3 +54,25 @@ You can see real examples for the following:
 
 * [Pure JS](https://github.com/CartoDB/viz-doc/tree/master/deck.gl/examples/pure-js): integrate in a pure js application, using webpack.
 
+
+### CARTO credentials
+
+CARTO credentials is an object to define the connection credentials:
```

Comment:
Maybe "This is an object to define the connection to CARTO, including the credentials (and optionally the parameters to point to specific api endpoints)"